wix-embedded-mysql/src/main/java/com/wix/mysql/MysqlClient.java

```diff
@@ -20,41 +23,60 @@
 
     private final MysqldConfig config;
     private final MysqldExecutable executable;
+    private final String schemaName;
 
-    public MysqlClient(
-            final MysqldConfig config,
-            final MysqldExecutable executable) {
+    public MysqlClient(final MysqldConfig config, final MysqldExecutable executable) {
+        this(config, executable, SystemDefaults.SCHEMA);
+    }
 
+    public MysqlClient(final MysqldConfig config, final MysqldExecutable executable, final String schemaName) {
         this.config = config;
         this.executable = executable;
+        this.schemaName = schemaName;
+    }
+
+    public void executeScripts(final List<File> files) {
+        for (File file: files) {
+            execute(format("source %s", file.getAbsolutePath()));
+        }
+    }
+
+    public void executeScripts(final File... files) {
+        for (File file: files) {
+            execute(format("source %s", file.getAbsolutePath()));
+        }
     }
 
-    public void apply(String sqlStatement) {
-        String statement = String.format((Platform.detect() == Windows) ? "\"%s\"" : "%s", sqlStatement);
+    public void executeCommands(final String... sqls) {
+        for (String sql: sqls) {
+            execute(sql);
+        }
+    }
 
+    private void execute(final String sql) {
+        String command = (Platform.detect() == Windows) ? format("\"%s\"", sql): sql;
```

Comment:
You mean why not use jdbc?

2 reasons:
 - it's already here and tested.
 - executing scripts using jdbc is not that easy, ex. http://stackoverflow.com/questions/1044194/running-a-sql-script-using-mysql-with-jdbc. So either I have to use spring or some other heavy dependency, or implement script processing myself. Then again mysql already provides this service, then why not just use it instead of reinventing the wheel.